### PR TITLE
20487D | Mod06 | LAK

### PR DESCRIPTION
**Pull request by VinayApposite**
CE complete. Comments inline.

### CODE REVIEW

File: Instructions/20487D_MOD06_LAK.md [hunk 1]
```diff
@@ -114,52 +113,52 @@
       }
     }
     ```
-    check **proxy_pass** this is the path of the revers proxy 
-    and the **test** is what need to be added for using **proxy_pass**
-5. Right click on **nginx** folder, select **New File** and name it **dockerfile**.
-6. Paste the following code to copy **nginx** image and change **conf** file:
+    check **proxy_pass** this is the path of the reverse proxy
```
Comment (VinayApposite) [inline Instructions/20487D_MOD06_LAK.md]: Should this line and the next be notes?

File: Instructions/20487D_MOD06_LAK.md [hunk 2]
```diff
@@ -326,214 +324,221 @@
...
 2. Navigate to **https://portal.azure.com**.
-3. If a page appears asking for your email address, enter your email address, and then click **Next** and enter your password, and then click **Sign In**.
-4. If the **Stay signed in?** dialog appears, click **Yes**.
+3. If a page appears asking for your email address, enter your email address, click **Next**, enter your password, and then click **Sign In**.
+4. If the **Stay signed in?** dialog box appears, click **Yes**.
    >**Note**: During the sign-in process, if a page appears prompting you to choose from a list of previously used accounts, select the account that you previously used, and then continue to provide your credentials.
-5. Click on **Create a resource** on the left menu panel.
-6. In the **Search** textbox type **API management** and click **Enter**, then click on **Create**.
+5. On the left menu panel, click **Create a resource**.
+6. In the **Search** box, type **API management**, click **Enter**, and then click **Create**.
```
Comment (VinayApposite) [inline Instructions/20487D_MOD06_LAK.md]: Should this be "press Enter"?

File: Instructions/20487D_MOD06_LAK.md (hunk 1 above)
Comment (slavBarkal) [reply Instructions/20487D_MOD06_LAK.md]: yes

File: Instructions/20487D_MOD06_LAK.md (hunk 2 above)
Comment (slavBarkal) [reply Instructions/20487D_MOD06_LAK.md]: yes

**Comment by barucht** [comment]
ready for QA

**Pull request merged**